cumulus/tasks/queue-granules/tests/index.js

```diff
@@ -14,16 +14,16 @@ const workflowTemplate = require('./fixtures/workflow-template.json');
 const aws = require('@cumulus/common/aws');
 
 test.beforeEach(async (t) => {
-  t.context.bucket = testUtils.randomString();
-  t.context.queue = testUtils.randomString();
-  await sqs().createQueue({ QueueName: t.context.queue }).promise();
+  t.context.bucket = randomString();
+  t.context.queueUrl = await createQueue();
   await s3().createBucket({ Bucket: t.context.bucket }).promise();
 });
 
-test.afterEach.always(async (t) => {
-  await recursivelyDeleteS3Bucket(t.context.bucket);
-  await sqs().deleteQueue({ QueueUrl: `http://${process.env.LOCALSTACK_HOST}:4576/queue/${t.context.queue}` }).promise();
-});
+test.afterEach.always(async (t) =>
+  Promise.all([
```

Comment:
I think the `await` is missing in front of `Promise.all`.

Reply:
Actually, the async is not needed.  Ava will handle the Promise that's being returned by Promise.all.